public/coda/styles.css

```diff
@@ -0,0 +1,18 @@
+.nav{
+    background: #ffff;
+    width: 100%;
+    max-width:initial;
+    height:80px;
+  }
+  .nav-link {
```

Comment:
Looks like this should have the same indentation as the others as on the same level?

Reply:
Noted. Thank you